### ISSUE TITLE
Light Mode

### ISSUE THREAD

**Issue by Sorena-29** [KudoAI/googlegpt#40]
In light mode, the font colour of the text box is also white making it extremely hard to see so I just use dark mode.

**Comment by adamlui** [comment]
hey @Sorena-29 that is odd, can you take a screenshot?

**Comment by Sorena-29** [comment]
I use Windows, Microsoft Edge, and Violentmonkey. Here is a screenshot:

![Screenshot 2024-06-27 230623](https://github.com/KudoAI/googlegpt/assets/174054216/1cd987f1-b431-4065-89ec-c529b1c1e4a8)

**Comment by adamlui** [comment]
IDK why I caxn't reproduce it, can you right-click on the answer > left-click 'Inspect', then in right panel, scroll down a few blocks until `#googlegpt > pre {`

![image](https://github.com/KudoAI/googlegpt/assets/10906554/85e2012d-0e8b-4c95-8b16-07ca40f95c43)

Is there a color property that's being striked thru?

**Comment by Sorena-29** [comment]
Sry for not getting back sooner, was busy with skl. I think u fixed it now. Light mode is now grey not white. I can see the text I type clearly now. Ty!

**Comment by Sorena-29** [comment]
Also would you recommend tampermonkey over violentmonkey?

**Comment by Sorena-29** [comment]
I couldn't find #googlegpt > pre {. I am a beginner at this stuff 😅. I attached a screen recording that might help.

https://github.com/KudoAI/googlegpt/assets/174054216/37645842-7ab5-412a-8746-302edf53470c

**Comment by adamlui** [comment]
@Sorena-29 that's strange, did you right-click exactly in the answer bubble when inspecting? Also somewhat related, what do you think of the latest dark scheme updates with stars in the background?

**Comment by Sorena-29** [comment]
I like the dark mode updates, I think it looks really cool. But I give up, I don't really know what I'm doing in the inspect thing. Can I just give you control of my laptop? Yk how when u contact Microsoft support and then they control ur laptop, can u do that as well? Then you can inspect anything u need. I'd be happy for you to do that.

**Comment by Sorena-29** [comment]
But yeah, I did right click exactly in the type bar.

**Comment by adamlui** [comment]
Oop;s sorry I meant the AI answer to you. If it still doesn't show up then we can do the remote inspection

**Comment by Sorena-29** [comment]
Oh, this?
![Screenshot 2024-07-08 134252](https://github.com/KudoAI/googlegpt/assets/174054216/9dd2b4b9-4e6e-42ff-bed4-c44b331caa64)

**Comment by adamlui** [comment]
Yes that is it, I notice your console shows in dark mode so something is making everything dark is the cause. Are you using a dark extension?

**Comment by Sorena-29** [comment]
I have something called 'Dark Reader' but it's turned off, I don't use it. I can delete it. I have dark mode turned on in my laptop settings because I think it looks better. I don't think dark mode is the cause tho. Surely something is causing the font to stay white when I switch to light mode in googleGPT, no? Because when I want to type something into the box in light mode, the font is white.
